pyston/serializer.py

```diff
@@ -350,7 +354,11 @@ def _get_verbose_value(self, raw, field_or_method, obj, **kwargs):
             return raw
 
     def _value_to_raw_verbose(self, val, field_or_method, obj, **kwargs):
-        return RawVerboseValue(val, self._get_verbose_value(val, field_or_method, obj, **kwargs))
+        return RawVerboseValue(
+            val,
+            self._get_verbose_value(val, field_or_method, obj, **kwargs),
+            data_to_python_callback=self._data_to_python
```

Comment:
whole problem is that RawVerboseSerializer should use method _get_serializer from source of RawVerboseValue object. _data_to_python firstly find the right serializer (with using typemapper) and serialize data. But if data is serialized in  RawVerboseSerializer doesn't have the right typemapper. Therefore RawVerboseValue must contain callback to the original _data_to_python method of source serializer class